ui/packages/catalog-ui-search/src/main/webapp/component/property/property.view.js

```diff
@@ -109,6 +111,12 @@ define([
           this.model.hasConflictingDefinitions()
         )
       },
+      handleCalculatedType: function() {
+        const input = this.$el.find('input')
+        if (input !== undefined) {
```

Comment:
✏️ I think this check could be simplified to. Doing this will evaluate to true if `input` does not equal `null`, `undefined`, `NaN`, `""`, `0`, or `false`

```
if (input) {
    . . .
}
```

Reply:
done